doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -1307,6 +1309,57 @@ Displaying information about scopes
       It also displays the delimiting key if any and the class to which the
       scope is bound, if any.
 
+Impact of scopes on printing
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When several notations are available for printing the same expression,
+Coq will use the following rules for printing priorities:
+
+- If two notations are available in different scopes which are open,
+  the notation in the more recently opened scope takes precedence.
+
+- If two notations are available in the same scope, the more recently
+  defined (or imported) notation takes precedence.
+
+- Abbreviations and lonely notations, both of which have no scope,
+  take precedence over a notation in an open scope if and only if the
+  abbreviation or lonely notation was defined (or imported) more
+  recently than when the corresponding scope was open. They take
+  precedences over any notation not in open scope, whether this scope
+  has a delimiter or not.
+
+- A scope is active for printing a term either because it was opened
```

Comment:
Perhaps emphasize "active" (or "active for printing a term") with italics, to indicate that this is the definition of that phrase.